dozer-sql/src/pipeline/expression/python_udf.rs

```diff
@@ -15,7 +15,7 @@ pub fn evaluate_py_udf(
     schema: &Schema,
     name: &str,
     args: &[Expression],
-    return_type: &FieldType,
+    return_type: &Option<FieldType>,
```

Comment:
What's the purpose of Option here, is this the return type of the UDF?
Do we allow to don't specify the return type? how it will works?

Reply:
Yes, we support python udf has no return value. Such as:

```python
def hello():
    print("hello")
```